.github/renovate.json5

```diff
@@ -0,0 +1,18 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["github>Hapag-Lloyd/Renovate-Global-Configuration"],
+
+  packageRules: [
+    {
+      matchManagers: ["github-actions"],
+      automerge: false,
+      description: "Do not auto-merge changes in the github-actions as we are referencing theses actions in the `this_*` files which would then create a new PR (circular dependency).",
+    },
```

Comment:
Does this rule make sense here? Since this is not a repository which contains shared workflows, this rule should not be required because there are no `this_*`-files, right?

Reply:
Makes no sense here.

The other rule has to be removed as well, as it is part of the default settings.